services/oauth_helpers.py

```
@@ -0,0 +1,186 @@
+
+    Raises:
+        ValueError: If credentials not configured or code exchange fails
+    """
+    client_id = client_id or os.getenv("OAUTH_CLIENT_ID")
+    client_secret = client_secret or os.getenv("OAUTH_CLIENT_SECRET")
+
+    if not client_id or not client_secret:
+        raise ValueError(
+            "OAUTH_CLIENT_ID and OAUTH_CLIENT_SECRET must be configured "
+            "in environment variables"
+        )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)